cmd/root.go

```diff
@@ -10,6 +10,9 @@ var rootCmd = &cobra.Command{
 	Use:   "note-cli",
 	Short: "note-cli is a CLI command tool for creating, writing, and managing note articles",
 	Long:  `note-cli is a CLI command tool for creating, writing, and managing note articles`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return cmd.Help()
+	},
```

Comment:
Hi, @aqyuki !! Thanks for Pull Request😄
How does  this change the display when executing command ```note-cli```?

I thought ```*cobra.Command.Help()``` was called by default if not set ```Run(or RunE)```.

Reply:
The modified output is as follows

```txt
note-cli is a CLI command tool for creating, writing, and managing note articles

Usage:
  note-cli [flags]
  note-cli [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  create      Create a new article directory.
  help        Help about any command

Flags:
  -h, --help     help for note-cli
  -t, --toggle   Help message for toggle

Use "note-cli [command] --help" for more information about a command.
```
The output is the same, but it is better to change it this way because explicitly calling it makes it obvious that the intended behavior is to display help if no sub command is specified.